inc/hclib-async.hpp

```diff
@@ -100,19 +99,19 @@ struct LambdaFutureWrapper<T, void> {
     }
 };
 
-/*
- * Yes, the name "async_at_hpt" sounds weird
- * but using this name to differentiate with the inter-node
- * "asyncAt" in HabaneroUPC++. Feel free to give a better
- * name to async_at_hpt.
- */
 template <typename T>
 inline void async(T &&lambda) {
     MARK_OVH(current_ws()->id);
     typedef typename std::remove_reference<T>::type U;
     hclib_async(lambda_wrapper<U>, new U(lambda), nullptr, nullptr, nullptr, 0);
 }
 
+/*
```

Comment:
I left the comment because it explains the rational weird name: `async_at_hpt`. I think we should leave it until we do something about the confusing name. The "at_hpt" suffix makes it sound like it takes an HPT argument rather than a place.